### PR TITLE
added new paragraph

### PR DESCRIPTION
**Pull request by AdiZurek**
I add new paragraph in index

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,6 +23,10 @@ <h3>jeszcze niższego poziomu nagłówek</h3>
         ad? Nam recusandae, consectetur doloremque maiores id ullam laudantium corrupti iure tenetur eveniet, officia
         voluptates!</p>
 
+    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Sint, distinctio cumque? Architecto, inventore explicabo
```
Comment (maxnatalia) [inline index.html:26]: I think it's great idea!

File: index.html [hunk 2]
```diff
@@ -23,6 +23,10 @@ <h3>jeszcze niższego poziomu nagłówek</h3>
         ad? Nam recusandae, consectetur doloremque maiores id ullam laudantium corrupti iure tenetur eveniet, officia
         voluptates!</p>
```
Comment (maxnatalia) [inline index.html:25]: I think, here have to change this paragraph

File: index.html (hunk 1 above)
Comment (AdiZurek) [reply index.html:26]: Thanks!

File: index.html (hunk 2 above)
Comment (AdiZurek) [reply index.html:25]: Yes, You are right!

File: index.html [hunk 3]
```diff
@@ -23,6 +23,10 @@ <h3>jeszcze niższego poziomu nagłówek</h3>
...
 
+    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Sint, distinctio cumque? Architecto, inventore explicabo
+        ut fuga reprehenderit amet eos ipsam laborum tempore totam molestiae officiis asperiores a, debitis nam
```
Comment (maxnatalia) [inline index.html:27]: Good job!

File: index.html [hunk 4]
```diff
@@ -23,6 +23,10 @@ <h3>jeszcze niższego poziomu nagłówek</h3>
...
+        adipisci?</p>
+
```
Comment (maxnatalia) [inline index.html:29]: I think this is all for now

**Review by maxnatalia — APPROVED** [review]
Excellent!

**Pull request closed without merging**